internal/datanode/data_sync_service.go

```diff
@@ -115,16 +115,19 @@ func newParallelConfig() parallelConfig {
 // start starts the flowgraph in datasyncservice
 func (dsService *dataSyncService) start() {
 	if dsService.fg != nil {
-		log.Debug("Data Sync Service starting flowgraph")
+		log.Debug("dataSyncService starting flowgraph", zap.Int64("collectionID", dsService.collectionID),
```

Comment:
There's no need to write log like 'dataSyncService'.  IMHO, 'data sync service' or 'datasync service' is better.

Reply:
ok. modify in next pr.